.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Integration
+
+on: [push, pull_request]
```

Comment:
Do we need the `pull_request` here? Isn't it implicit on push? :thinking:

Reply:
It's not, push only happens on branches that belong to this repo, but not on forks. If we want the CI to run for external contributions we need pull_request

Reply:
Right! Noice